frappe/public/js/frappe/form/grid.js

```diff
@@ -136,19 +136,24 @@ export default class Grid {
 			var dirty = false;
 
 			let tasks = [];
-			me.deleted_docs = [];
-
 			me.get_selected().forEach((docname) => {
 				tasks.push(() => {
 					if (!me.frm) {
-						me.deleted_docs.push(docname);
```

Comment:
Why remove `deleted_docs`? Isn't it used somewhere?

Reply:
In my testing I didn't see it being used anywhere except for this place where it was broken anyway. 
In this particular case, the `if` block it is in, only works for webforms and not on the desk. So event in the worst case scenario, the desk stays intact.